api/v1/api.proto

```diff
@@ -0,0 +1,395 @@
+        bytes manifest = 6;
+}
+
+// KeyValue is the type used by annotations and labels in PodFilter and ImageFilter.
+message KeyValue {
+        // Key part of the key-value pair.
+        string Key = 1;
+        // Value part of the key-value pair.
+        string value = 2;
+}
+
+// PodFilter defines the condition that the returned pods need to satisfy in ListPods().
+// The conditions are combined by 'AND'.
+message PodFilter {
+        // If not empty, then the returned pods must be in one of these states.
+        repeated PodState states = 1;
+
+        // If not empty, then the returned pods must have one of these names in the apps.
+        repeated string app_names = 2;
+
+        // If not empty, then the returned pods must contain at least one of these image names in the apps.
+        repeated string image_names = 3;
+
+        // If not empty, then the returned pods must be in at least one of these networks.
+        repeated string network_names = 4;
+
+        // If not empty, then the returned pods must contain all the listed annotations.
+        repeated KeyValue annotations = 5;
+}
+
+// ImageFilter defines the condition that the returned images need to satisfy in ListImages().
+// The conditions are combined by 'AND'.
+message ImageFilter {
+        // If not empty, then the returned images must have the listed ids.
+        repeated string ids = 1;
+
+        // if not empty, then the returned images must have one of the listed prefixes.
+        repeated string prefixes = 2;
+
+        // If not empty, then the returned images must contain one of the listed names as the base name.
+        // For example, both 'coreos.com/etcd' and 'k8s.io/etcd' will be returned if 'etcd' is included,
+        // however 'etcd-backup' will not be returned.
+        repeated string base_names = 3;
+
+        // If not empty, then the returned images' names must contain one of the listed keywords.
+        // For example, both 'kubernetes-etcd', 'etcd:latest' will be returned if 'etcd' is included,
+        repeated string keywords = 4;
+
+        // If not empty, then the returned images must have one of the listed labels.
+        repeated KeyValue labels = 5;
+
+        // If set, then the returned images must be imported after this timestamp.
+        int64 imported_after = 6;
+
+        // If set, then the returned images must be imported before this timestamp.
+        int64 imported_before = 7;
+
+        // If not empty, then the returned images must contain all the listed annotations.
+        repeated KeyValue annotations = 8;
+}
+
+// Info describes the information of rkt on the machine.
+message Info {
+        // Version of rkt, required, in the form of Semantic Versioning (http://semver.org/).
+        string rkt_version = 1;
+
+        // Version of appc, required, in the form of Semantic Versioning (http://semver.org/).
+        string appc_version = 2;
```

Comment:
Should the version of this event stream protocol be added there? `v1` at the moment...